frontend/pages/collections/[slug].page.tsx

```diff
@@ -78,3 +103,113 @@ export const getStaticPaths: GetStaticPaths = async () => {
     fallback: "blocking",
   };
 };
+
+const MobileViewComponent: FC<CollectionProps> = ({ collection }: CollectionProps) => {
+  const [showIcons, setShowIcons] = useState(false);
+  const onClick = () => {
+    setShowIcons(true);
+    if (showIcons) {
+      setShowIcons(false);
+    }
```

Comment:
`setState` can also take a function that has the previous state as the parameter, so this can be simplified to 
```jsx
setShowIcons((prevState) => !prevState)
```